packages/runtime-common/search-index.ts

```diff
@@ -235,6 +236,12 @@ export class SearchIndex {
     return { type: 'doc', doc };
   }
 
+  // this is for tests--it's unlikely we'd actually want to access CSS directly
+  // this way
+  async css(url: URL, opts?: Options): Promise<IndexedCSSOrError | undefined> {
```

Comment:
I couldn’t find this being used anywhere but if it’s test-specific could this package have a test helpers file instead?